proxy/hdrs/HTTP.h

```diff
@@ -758,7 +767,7 @@ HTTPVersion::operator>(const HTTPVersion &hv) const
 inline int
 HTTPVersion::operator<(const HTTPVersion &hv) const
 {
-  return (m_version < hv.m_version);
+  return vmajor < hv.get_major() || (vmajor == hv.get_major() && vminor < hv.get_minor());
```

Comment:
Wait - this is what I recommended on the previous method. Might be worth it to not re-use, but ultimately I suspect it doesn't matter.